sys-devel/native-cctools/native-cctools-5.ebuild

```diff
@@ -39,18 +37,9 @@ src_install() {
 		*-solaris*)
 			nativepath=/usr/sfw/bin
 		;;
-		*-aix*)
-			nativepath=/usr/ccs/bin
-			wrappers=("${wrappers[@]}" "ld=${FILESDIR}/${AIX_V}/ld")
-			wrappers=("${wrappers[@]}" "nm=${FILESDIR}/${AIX_V}/nm")
-			wrappers=("${wrappers[@]}" "mkexpfile=${FILESDIR}/${AIX_V}/mkexpfile")
-		;;
 		*-apple-darwin*|*-netbsd*|*-openbsd*)
```

Comment:
what about the non-apple stuff?

Reply:
Yeah, fair point. Any port to *BSD would need to be done entirely from scratch.